modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -315,42 +303,7 @@ private void _validateAPISchema(
 		}
 	}
 
-	private void _validateRegularPath(
-			ObjectEntry objectEntry, String pathString)
-		throws Exception {
-
-		Matcher matcher = _pathPattern.matcher(pathString);
-
-		if (!matcher.matches()) {
-			User user = _userLocalService.getUser(objectEntry.getUserId());
-
-			ObjectField objectField = _objectFieldLocalService.getObjectField(
-				objectEntry.getObjectDefinitionId(), "path");
-
-			String message =
-				"%s can have a maximum of 255 alphanumeric characters";
-			String messageKey =
-				"x-can-have-a-maximum-of-255-alphanumeric-characters";
-
-			if (!pathString.startsWith(StringPool.FORWARD_SLASH)) {
-				message = "%s must start with the \"/\" character";
-				messageKey = "x-must-start-with-the-x-character";
-			}
-
-			if (!StringUtil.isLowerCase(pathString)) {
-				message = "%s must contain only lower case characters";
-				messageKey = "x-must-contain-only-lower-case-characters";
-			}
-
-			String label = objectField.getLabel(user.getLocale());
-
-			throw new ObjectEntryValuesException.InvalidObjectField(
-				Arrays.asList(label, "\"/\""), String.format(message, label),
-				messageKey);
-		}
-	}
-
-	private void _validateSingleElementPath(
```

Comment:
I think it should name as the `_validateSingleElementPath` because it could be used in future endpoints